.github/workflows/build-and-release-edge.yml

```diff
@@ -27,10 +27,13 @@ jobs:
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Docker edge build & tag
         run: |
-          DOCKER_BUILDKIT=1 docker build -f install/docker/Dockerfile --no-cache -t ${{ secrets.IMAGE_NAME }}:edge-latest --build-arg TOKEN=${{ secrets.GLOBAL_TOKEN }} --build-arg GIT_COMMITSHA=${GITHUB_SHA::8} --build-arg GIT_VERSION="edge-latest" --build-arg RELEASE_CHANNEL=${RELEASE_CHANNEL} .
```

Comment:
The `edge-latest` still gets included, right? We're just adding a release number tag, too.

Reply:
Can you offer an example of what the "release number" will be for an edge release? v0.6.99-1? Or are we going to keep overwriting the same tag `edge-v0.6.99` until a new stable release is made?

Reply:
I see that the former is true.